### PR TITLE
fix: correct BatchNorm parameter name for PyTorch compatibility

### PR DESCRIPTION
**Pull request by Illustracer**
PaddleOCR v4 和 v5 的 mobile 识别模型使用的 backbone PPLCNetV3 在重参数化过程中存在参数转换错误。具体表现为在 LearnableRepLayer._get_kernel_bias() 方法中，代码尝试访问 _epsilon, _mean, _variance 等参数

The backbone PPLCNetV3 used in PaddleOCR v4 and v5 mobile recognition models has parameter conversion errors during reparameterization. Specifically, the LearnableRepLayer._get_kernel_bias() method attempts to access branch.bn._epsilon, _mean, _variance, etc

### CODE REVIEW

**Comment by Illustracer** [comment]
#104
